roofit/roofitcore/src/TestStatistics/RooSumL.cxx

```diff
@@ -0,0 +1,90 @@
+// Author: Patrick Bos, Netherlands eScience Center / NIKHEF 2021
+
+/*****************************************************************************
+ * RooFit
+ * Authors:                                                                  *
+ *   WV, Wouter Verkerke, UC Santa Barbara, verkerke@slac.stanford.edu       *
+ *   DK, David Kirkby,    UC Irvine,         dkirkby@uci.edu                 *
+ *                                                                           *
+ * Copyright (c) 2000-2021, Regents of the University of California          *
+ *                          and Stanford University. All rights reserved.    *
+ *                                                                           *
+ * Redistribution and use in source and binary forms,                        *
+ * with or without modification, are permitted according to the terms        *
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)             *
+ *****************************************************************************/
+
+#include <TestStatistics/RooSumL.h>
+#include <RooAbsData.h>
+#include <TestStatistics/RooSubsidiaryL.h>
+
+#include <algorithm> // min, max
+
+namespace RooFit {
+namespace TestStatistics {
+
+// Note: components must be passed with std::move, otherwise it cannot be moved into the RooSumL because of the unique_ptr!
```

Comment:
`//` --> `///`, and extend the docs. Doxygen actually supports `\note <text>`, so you get colours and all.